### PR TITLE
Fix engage request timeout increase

### PR DESCRIPTION
**Pull request by oquerne**

### CODE REVIEW

File: deepomatic/cli/lib/platform.py [hunk 1]
```diff
@@ -13,11 +13,14 @@
 
 
 LOGGER = logging.getLogger(__name__)
+REQUEST_TIMEOUT = float(os.getenv("REQUEST_TIMEOUT", "40."))
```
Comment (maingoh) [inline deepomatic/cli/lib/platform.py:16]: nit: maybe add an `S` at request to remind it is related to the `requests` package

File: deepomatic/cli/common.py [hunk 2]
```diff
@@ -19,6 +19,7 @@
 SUPPORTED_PROTOCOLS_INPUT = ['rtsp', 'http', 'https']
 SUPPORTED_IMAGE_OUTPUT_FORMAT = SUPPORTED_IMAGE_INPUT_FORMAT
 SUPPORTED_VIDEO_OUTPUT_FORMAT = ['.avi', '.mp4']
+REQUESTS_DEFAULT_TIMEOUT = float(os.getenv("REQUEST_TIMEOUT", "40."))
```
Comment (maingoh) [inline deepomatic/cli/common.py:22]:
```suggestion
REQUESTS_DEFAULT_TIMEOUT = float(os.getenv("REQUESTS_TIMEOUT", "40."))
```

File: deepomatic/cli/lib/platform.py [hunk 3]
```diff
@@ -16,8 +17,10 @@
 
 
 class DrivePlatformManager(object):
-    def __init__(self, client_cls=HTTPHelper):
-        self.drive_client = client_cls()
+    def __init__(self):
+        self.drive_client = HTTPHelper(
+            requests_timeout=REQUESTS_DEFAULT_TIMEOUT
```
Comment (maingoh) [inline deepomatic/cli/lib/platform.py:22]: Can you take the opportunity to add the user agent in all those places as well

File: tests/test_site.py [hunk 4]
```diff
@@ -140,7 +140,8 @@ def tmp():
 @contextmanager
 def setup():
     tmp_dir = tempfile.mkdtemp()
-    manager = SiteManager(path=tmp_dir, client_cls=MockApi)
+    manager = SiteManager(path=tmp_dir)
+    manager._client = MockApi()
```
Comment (maingoh) [inline tests/test_site.py:144]: Actually maybe for the site_manager we should keep the `client_cls` (if the HTTPHelper fails to instanciate before patching it)

**Review by maingoh — APPROVED** [review]
Nice !!

File: tests/test_site.py (hunk 4 above)
Comment (maingoh) [reply tests/test_site.py:144]: Or use `unittest.patch` to replace `HTTPHelper` class with `MockApi`

**Review by alemoreau — APPROVED** [review]

**Pull request merged**